tests/SprykerEcoTest/Zed/Unzer/Business/CheckIsAuthorizeCanceledOmsConditionTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Unzer\Business;
+
+use Generated\Shared\DataBuilder\SaveOrderBuilder;
+use Generated\Shared\Transfer\PaymentUnzerOrderItemTransfer;
+use Generated\Shared\Transfer\PaymentUnzerTransfer;
+use SprykerEco\Zed\Unzer\UnzerConstants;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerTest
+ * @group Zed
+ * @group Unzer
+ * @group Business
+ * @group Facade
+ * @group CheckIsAuthorizeCanceledOmsConditionTest
+ */
+class CheckIsAuthorizeCanceledOmsConditionTest extends UnzerFacadeBaseTest
+{
+    /**
+     * @var string
+     */
+    public const STATE_MACHINE_PROCESS_NAME = 'UnzerSofort01';
+
+    /**
+     * @return void
+     */
+    public function testCheckIsAuthorizeCanceledOmsConditionReturnsTrueWhileOmsStatusIsAuthorizeCanceled(): void
+    {
+        //Arrange
```

Comment:
```suggestion
        // Arrange
```

same everywhere as I asked before

Reply:
Fixed.